app/src/main/java/io/github/androidacademyglobal/ui/theme/Constants.kt

```diff
@@ -1,6 +1 @@
-package io.github.androidacademyglobal.ui.theme
-
-const val FONT_SIZE_REGULAR = 16
-
-const val MARGIN_TINY = 4
-const val ZERO = 0
\ No newline at end of file
+package io.github.androidacademyglobal.ui.theme
```

Comment:
if this file becomes empty, maybe there is a point in dropping it?

Reply:
It is empty now, so I left it there